core/profile.c

```diff
@@ -642,7 +643,20 @@ static void populate_plot_entries(struct dive *dive, struct divecomputer *dc, st
 	/* Add two final surface events */
 	plot_data[idx++].sec = lasttime + 1;
 	plot_data[idx++].sec = lasttime + 2;
+
+	/* We probably used less plot_data entries than needed.
```

Comment:
my favorite pet peeve in the English language. If you can count it, it's 'fewer', otherwise less. I give you less respect, but fewer dollars.